### PR TITLE
Adds detection for KRIP brand, improves detection for Alcatel, Blu, BMobile, Epik One, Gateway, Huawei, Hyundai, IKU Mobile, MobiWire, MicroMax, Positivo, Safaricom, TCL, UMIDIGI, Vivo brands and improves detection for Opera GX, Windows, x86, x64 platforms

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -25111,6 +25158,14 @@ KREZ:
       device: 'tablet'
       model: '$1 3G'
 
+# KRIP (https://www.krip.com/)
+KRIP:
+  regex: 'K4m'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:25163]: regex: 'K4m(?:[);/ ]|$)'

File: regexes/client/browsers.yml [hunk 2]
```diff
@@ -1261,7 +1261,7 @@
   version: ''
 
 # Opera GX (https://www.opera.com/gx#start | http://ftp.opera.com/pub/opera_gx/)
-- regex: 'OPRGX(?:/(\d+[\.\d]+))?'
+- regex: 'OP(?:RG)?X(?:/(\d+[\.\d]+))?'
```
Comment (sanchezzzhak) [inline regexes/client/browsers.yml:1264]: https://apps.apple.com/en/app/opera-gx/id1559740799

* Mozilla/5.0 (iPhone; CPU iPhone OS 14_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1

* Mozilla/5.0 (iPad; CPU OS 15_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1

What type of engine will we get if it's an Iphone and iPad

Comment (liviuconcioiu) [reply regexes/client/browsers.yml:1264]: Blink

Comment (sanchezzzhak) [reply regexes/client/browsers.yml:1264]: Modify current rule to
```yml 
regex: 'Chrome/.+ OP(?:RG)?X(?:/(\d+[\.\d]+))?'
# ...
   engine: Blink
```
Create next rule
```yml

regex:  'OP(?:RG)?X(?:/(\d+[\.\d]+))?'
# ...
   engine: WebKit
 ```
And added test for IOS~
* Mozilla/5.0 (iPad; CPU OS 15_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**